smtp/client.js

```diff
@@ -85,7 +85,7 @@ Client.prototype =
       if(!(msg instanceof message.Message) 
           && msg.from 
           && (msg.to || msg.cc || msg.bcc)
-          && (msg.text || this._containsInlinedHtml(msg.attachment)))
+          && ((msg.text || '' != null) || this._containsInlinedHtml(msg.attachment)))
```

Comment:
`'' ~= null` will always return true. instead you would do `msg.text != undefined` if you want to just test if msg has a field of `text`